web.go

```diff
@@ -25,12 +25,13 @@ var metrics = stats.New()
 // WebProvider is a provider.Provider implementation that provides the UI.
 // FIXME to be handled another way.
 type WebProvider struct {
-	Address  string `description:"Web administration port"`
-	CertFile string `description:"SSL certificate"`
-	KeyFile  string `description:"SSL certificate"`
-	ReadOnly bool   `description:"Enable read only API"`
-	server   *Server
-	Auth     *types.Auth
+	Address    string `description:"Web administration port"`
+	CertFile   string `description:"SSL certificate"`
+	KeyFile    string `description:"SSL certificate"`
+	ReadOnly   bool   `description:"Enable read only API"`
+	Statistics bool   `description:"Enable more detailed statistics"`
```

Comment:
perhaps if it ended up being exposed as statistics.recent_errors that would be nice?